### PR TITLE
Lifestyle

### PR DESCRIPTION
**Pull request by donnell9**
Kind of janky way of integrating the data (used if statements) but it worked, not sure if there's a cleaner way using the map function

### CODE REVIEW

File: src/data/data_surveys.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        advice: 'Just like practicing your ollies, you can only get good at ECE if you practice. You may not like it, but that is what peak success in ECE looks like.',
+        fav_project: '',
+        fav_class: '',
+        why_ece: ''
+    },
+    {
+        name: 'Matthew Wen',
+        fun_fact: 'I spend around like 200 hours a year on Spotify. Such a great rate of return.',
+        advice: 'There are nice people in ECE. Avoid the people with superiority complexes.',
+        fav_project: '',
```
Comment (matthewwen) [inline src/data/data_surveys.tsx]: If it is an undefined property, you can just delete it. Javascript can handle the rest.

File: src/pages/ece.tsx [hunk 2]
```diff
@@ -17,6 +18,101 @@ function Ece(props) {
...
+                { children }
+            </div>
+        )
+    }
+}
+
+function Advice(props) {
+    const {title, author, children} = props;
+    if (author.advice === ''){
+        return null
```
Comment (matthewwen) [inline src/pages/ece.tsx]: maybe <></> is better. I'm not sure.

File: src/data/data_surveys.tsx [hunk 3]
```diff
@@ -0,0 +1,219 @@
+const SURVEYDATA = [
+    {
+        name: 'Sara Hui',
+        fun_fact: 'From an island, not a good swimmer. iykyk.',
+        advice: 'Take your time and don’t rush through classes. It’s more enjoyable if you don’t cram all your core classes in one semester.',
+        fav_project: 'Optical Heartbeat Sensor in 20008. I used in infrared sensor for heartbeat detection, a couple of filters to even the signal out, inverters, and comparators to light up an LED for each heartbeat. It was suuper cool!',
```
Comment (matthewwen) [inline src/data/data_surveys.tsx:6]: It looks good, but you feel like you are limited to just {attribute: string value}. You can do something like this
fav_project: {
 name: 'name',
 date: 'date',
 etc...
}

Comment (donnell9) [reply src/data/data_surveys.tsx:6]: That's a good idea. Might need to send out the survey again so people can fill out whatever attributes we choose if we want to go that route

**Pull request merged**